Day-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,8 @@
 import React from 'react';
 import './Button.css';
+
+function ActionButton(props){
+    return <div className={props.actionButton.style}>{props.actionButton.text}</div>
```

Comment:
This looks like a good opportunity for [object destructuring]() for making your code less repetitive and a bit easier to read as well. For example:
```javascript
function ActionButton(props){
    const { style, text } = props.actionButton;
    return <div className={style}>{text}</div>;
}
```
or even:
```javascript
function ActionButton({ actionButton }){
    const { style, text } = actionButton;
    return <div className={style}>{text}</div>;
}
```